### DISCUSSION TITLE
Feature Request:  SOC Web interface IDS Rule (modify/disable/update) Integration

### DISCUSSION THREAD

**Discussion by xfaith** [Security-Onion-Solutions/securityonion#7939] [Ideas] [answered]
I wanted to make a suggestion/feature request for the SOC.

I would like to see a link in the left-hand pane of the SOC for RuleSets which could bring up a GUI front end for Suricata rules (or all the ruleset)  Where you could enable/disable/modify/create new, and conduct rule-updates from the SOC.   (Could be user-based access to this area).  I have certain users who I would rather not have full access to the CLI, but they would still need to update rules, suppress/threshold/modify ect. I think by containing them to the SOC it would be a better overall experience and lessen the chances of issues by users who don't understand CLI and cause system failures.   

I know there are things like Scirius (it looks like it has CyberChef, and possibly other applications)  not sure how well that would play even.

**Comment by dougburks** [comment, marked as answer]
Yes, we have plans to add this capability to SOC.
